src/ziggurat/streams.clj

```diff
@@ -284,3 +282,26 @@
              (start-streams (:stream-routes (mount/args)) (ziggurat-config)))
   :stop (do (log/info "Stopping Kafka streams")
             (stop-streams stream)))
+
+(defn add-stream-thread
+  [stream]
```

Comment:
It should take a stream-config-id (aka topic-entity) as an input and internally we can fetch the stream object from streams state.

Reply:
done